src/main/java/com/productdock/library/inventory/adapter/in/web/BookSubscriptionApi.java

```diff
@@ -0,0 +1,43 @@
+package com.productdock.library.inventory.adapter.in.web;
+
+import com.productdock.library.inventory.application.port.in.BookSubscriptionUseCase;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.oauth2.jwt.Jwt;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("api/inventory/subscriptions")
+@Slf4j
+public record BookSubscriptionApi(BookSubscriptionUseCase bookSubscriptionUseCase) {
+
+    public static final String EMAIL = "email";
```

Comment:
I'm not sure if you already have some agreements on this, but I suggest adding a todo comment whenever we use hardcoded values, to indicate to the reviewers that this is temporarily